de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1193,9 +1193,70 @@ Now, in the _Navigator_ double click on ConfigurationTree and scroll to the Mass
+
+There are _InterfaceEnds_ that are of a specific _InterfaceType_  and have the potential to connect _Interfaces_.
+
+_InterfaceTypes_  need to be defined as child elements of an _InterfaceTypeCollection_ .
+The _InterfaceTypeCollection_  is created by clicking with the right mouse on the system model and select menu:Context[Functional Electrical Architecture>Add InterfaceTypeCollection].
+image:chapterModeling/subsectionFEA/InterfaceType.png[InterfaceType_Collection]
+A type can be created by clicking with the right mouse on the _InterfaceTypeCollection_ and selecting menu:Context[Functional>Electrical Architecture>Add InterfaceType]
+image:chapterModeling/subsectionFEA/AddInterfaceType.png[InterfaceType]
+
+An _InterfaceType_ can be a _DataInterfaceType_ , a _ThermalReferencePoint_ or a _PowerInterfaceType_.
+
+By adding a _RangedVoltageDefinition_ or a _FixedVoltageDefinition_ to the _PowerInterfaceType_ a minimum and maximum voltage  can be specified.
+
+image:chapterModeling/subsectionFEA/VoltageSpecificationPowerInterfaceType.png[Specify the voltage of a power Interface]
+
+_InterfaceEnds_  should be defined in a product tree as this definition enables elements of the configuration tree to inherit the properties defined inside the product tree. It is added to the product tree by clicking with the right mouse on the menu:Context[product tree >Functional Electrical Architecture>Add _Interface_End].
 
+An _Interface_ can be added  manual to a configuration tree by using the right mouse and navigating to menu:Context[Functional Electrical Architecture>Add Interface].
+The end and start point of the _Interface_ can be set as properties of the _Interface_ as shown in the following image:
+image:chapterModeling/subsectionFEA/InterfaceEnd.PNG[_Interface_End]
+
+Additionally it is possible to add an _Interface_ as described in the following subsection.
+
+===== Add a new diagram
+:step: 0
+[%header,cols=">8,62a,40a"] 
 |===
 
+|Step
+|Task
+|Image
+
+|{counter:s}.
+|Click on the documents folder in the *ConfigurationTree*.
+|image:chapterModeling/subsectionFEA/DocumentsElement.PNG[DE]
+
+|{counter:s}.
+|Click on the new elements symbol.
+|image:chapterModeling/subsectionFEA/NewElement.PNG[NE]
+
+|{counter:s}.
+|Choose menu:Context[VirSat>Virsat Diagram].
+|image:chapterModeling/subsectionFEA/Diagram.PNG[NE]
+
+|{counter:s}.
+|Choose _Interface_ in the btn:[Drop Down] menu.
+|
+
+|{counter:s}.
+|Drag and Drop the *Element Configurations* from the satellite navigator to the diagram editor .
+|
+
+|{counter:s}.
+|The _Interface_ can be used in the system diagram to connect system elements with a power supply by drag and dropping the _Interface_ to their _Interface_End.
+|image:chapterModeling/subsectionFEA/InterfaceInTheDiagram.PNG[InterfaceIn]
+|===
+
+The properties of the _Interface_ and the _InterfaceType_ can be imported and exported from an excel spreadsheet.

```

Comment:
Maybe we should have this in a subsection and describe how to do the import / export.

Reply:
Done Opened a new issue